docs/manual/docs/administrator-guide/managing-users-and-groups/index.md

```diff
@@ -76,3 +76,126 @@ Rights associated with the roles are illustrated in detail in the list below:
+    |---------------------------------------------| :---: |:-----------------------:| :---: | :---: |:-----------------------:|:-----------------------:|
+    | Metadata selection / Export (ZIP)           | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Export (PDF)           | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Export (CSV)           | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Selection only         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Update privileges      |                         |                         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Publish                |                         |                         | | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Unpublish              |                         |                         | | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Approve                |                         |                         | | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Transfer Ownership     |                         |                         | | | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Validate               |                         |                         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Validate records links |                         |                         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Updates categories     |                         |                         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Delete                 |                         |                         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Index records          |                         |                         | | | :material-check-circle: | :material-check-circle: |
```

Comment:
For `UserAdmin` the options are there, but can't really do anything. Probably need to be updated the code to remove them, unless the user has also `Editor` / `Reviewer` profiles.

Not really a problem of the documentation, but of the application.